backend/copr_backend/actions.py

```diff
@@ -387,17 +387,16 @@ def run(self):
         self.log.info("Action delete project chroot.")
 
         ext_data = json.loads(self.data["data"])
-        ownername = ext_data["ownername"]
-        projectname = ext_data["projectname"]
-        chrootname = ext_data["chrootname"]
+        storage = storage_for_enum(ext_data["storage"], self.opts, self.log)
```

Comment:
This reminds me the note from previous PR, that having something like:
```
storage = CoprResultStorage(ext_data["ownername"], ext_data["project_name"])
storage.delete_repository(ext_data["dirname"])
```
Would be much more convenient.  Maybe even:

```
storage = copr.get_storage()
```